getting-started/getting-started-activiti-cloud.md

```diff
@@ -323,50 +292,48 @@ $ kubectl get services
 
 ![New Record Set in Route 53.](../.gitbook/assets/route-53-record-set-elb-dns.png)
 
-Open the “_values.yaml_” file located at: activiti-cloud-charts/activiti-cloud-full-example.
-
-Set the Activiti Cloud modeling to true and replace the 3 occurrences of the string “**REPLACEME**” to "**&lt;your-public-domain&gt;**". In our case: _**raphaelallegre.com**_ in every occurrence of “**REPLACEME**”.
-
-Example "_values.yaml_" file:
-
-```yaml
-# Default values for activiti-cloud-full-example.
-...
-
-global:
-  keycloak:
-    url: "http://activiti-keycloak.raphaelallegre.com/auth"
-  gateway:
-    host: &gatewayhost "activiti-cloud-gateway.raphaelallegre.com"
-
-activiti-cloud-modeling:
-  enabled: true
-...
-infrastructure:
-  activiti-keycloak:
-    keycloak:
-      enabled: true
-      keycloak:
-        ingress:
-          enabled: true
-          path: /
-          proxyBufferSize: "16k"
-          hosts:
-            - "activiti-keycloak.raphaelallegre.com"
- ...
-```
+We will use "**your-public-domain**" to deploy Activiti Helm chart in the next section. In our case: **raphaelallegre.com**.
 
 ### C\) Deploy the Helm chart
 
-Once you have performed the 4 changes, deploy the chart by running the following command:
+Once you have resolved you domain name, install Helm chart by running the Helm install command using your public domain name to set the `global.gateway.domain` key. In our case replace the string “**REPLACEME**” with the domain from previous step.
 
 ```bash
-$ helm install -f values.yaml activiti-cloud-charts/activiti-cloud-full-example
+$ helm install example activiti-cloud-charts/activiti-cloud-full-example --set global.gateway.domain=REPLACEME
```

Comment:
@igdianov here "example" is wrong I think and in order to deploy it in my cluster I've used: 
```
helm install activiti-cloud-charts/activiti-cloud-full-example --set global.gateway.domain=35.228.237.6.nip.io --namespace=salaboy
```

and that works like a charm!

Reply:
I think that we need something like -n for the name of the release, but we can leave it to the random option by default

Reply:
@salaboy I think it makes sense to use `--name` option instead of random name. I will update the guide.